src/main/java/com/epam/reportportal/annotations/attribute/Attributes.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2019 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.epam.reportportal.annotations.attribute;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * @author <a href="mailto:ivan_budayeu@epam.com">Ivan Budayeu</a>
+ */
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.METHOD)
+public @interface Attributes {
+
+	String[] component() default {};
+
+	String[] e2e() default {};
+
+	String[] persona() default {};
+
+	String[] product() default {};
+
+	String[] vertical() default {};
+
+	Attribute[] attributes() default {};
+
+	MultiKeyAttribute[] multiKeyAttributes() default {};
+
+	MultiValueAttribute[] multiValueAttributes() default {};
```

Comment:
I would not use 3 different fields here, but just one with `annotation.Annotation` type. Then in code you can filter out yours annotations. Because it is now three of them, but we can add more.

Reply:
Constants-related annotation fields removed (aka persona, product, etc.)